src/app/users/editPermission.js

```diff
@@ -56,7 +56,7 @@ const post = async (req, res) => {
       newValue: permissionName,
     }],
   });
-  res.flash('info', `${user.firstName} ${user.lastName} now has ${permissionName} access to ${organisationDetails.organisation.name}`);
+  res.flash('info', role === 10000 ? `${user.firstName} ${user.lastName} now has approver access to ${organisationDetails.organisation.name}` : `${user.firstName} ${user.lastName} approver access removed for ${organisationDetails.organisation.name}` );
```

Comment:
The 2 variations have different tense. The second option should probably be `${user.firstName} ${user.lastName} approver access has been removed for ${organisationDetails.organisation.name}`